executor/src/witgen/jit/single_step_processor.rs

```diff
@@ -324,12 +412,11 @@ VM::instr_mul[1] = 1;"
             Ok(_) => panic!("Expected error"),
             Err(e) => {
                 let expected = "\
-Main::is_arith $ [ Main::a, Main::b, Main::c ]
-     ???              2       ???      ???    
-                                              
-Main::is_arith        2     Main::b  Main::c  
-     ???                      ???      ???    
-                                          ";
+  Main::is_arith $ [ Main::a, Main::b, Main::c ]
```

Comment:
Not really sure why this is now indented.

Reply:
Ah, I used the same code as we have for the block machine, where it is indented as well.